src/stylesheets/main.scss

```diff
@@ -103,6 +103,7 @@ body {
   img {
     box-sizing: border-box;
     width: 100%;
+    max-width: 722px;
```

Comment:
This impacts images on other pages such as https://deploy-preview-970--govuk-design-system-preview.netlify.com/patterns/payment-card-details/ is this what we want?

Reply:
Good spot.

We probably want to look at updating https://github.com/alphagov/govuk-design-system/blob/5606c0ee17a2e3ec2caa41e76fe439ddcad89307/src/stylesheets/main.scss#L90-L100 if this is what we want?

Reply:
@36degrees @nickcolley Yep, hoping to do this to all images. I also put it in the Ethnicity pattern hoping that it would be merged first but snuck it in here.